packages/core/src/index.js

```diff
@@ -1,12 +1,12 @@
 import 'babel-polyfill';
 import 'promise-polyfill';
-import '_common/lib/polyfills/nodelist.foreach';
-import '_common/lib/polyfills/element.closest';
```

Comment:
are these polyfills not in used in trader anymore?

Reply:
@kelly-binary Nope, those are legacy from `binary-static` where we used them to directly modify the `Element`.